src/components/signon/SignOn.tsx

```diff
@@ -366,11 +366,9 @@ const SignOn = ({ navigation }: SignOnProps) => {
       setEmail('')
       setPassword('')
 
```

Comment:
Not sure if `signedIn` is true after sign up but I assume it is

Reply:
it is